react/src/Inspect.js

```diff
@@ -147,13 +147,23 @@ function PieceStatus({pieceCid, pieceStatus, searchQuery}) {
                 {searchIsPayloadCid ? (
                     <tr key="payload cid">
                         <th>Searched CID (non-root)</th>
-                        <td>{searchQuery}</td>
+                        <td>
+                            <span>{searchQuery}</span>
+                            <a className="download" target="_blank" href={"/download/block/"+searchQuery}>
+                                Download block
+                            </a>
+                        </td>
                     </tr>
                 ) : null}
                 {rootCid ? (
                     <tr key="data root cid">
                         <th>Data Root CID</th>
-                        <td>{rootCid}</td>
+                        <td>
+                            <span>{rootCid}</span>
+                            <a className="download" target="_blank" href={"/download/block/"+searchQuery}>
```

Comment:
After playing with this going over the booster-http updates locally, I am realizing this needs to be rootCid and not searchQuery `<a className="download" target="_blank" href={"/download/block/"+rootCid}>`